README.md

```diff
@@ -142,6 +142,25 @@ Request the IPFS node to pin the content hash.
 
 ---
 
+#### `ANNOUNCE_CLIENT`
```

Comment:
Is that really necessary?

Reply:
Oh good shout, that should have been removed.